hardwareModules/RoFICoM/software/control_board/vl53l1_lidar/lidar.hpp

```diff
@@ -0,0 +1,354 @@
+#pragma once
+
+// REMOVE: its it currently to enable assert in release
+#undef NDEBUG
+#include <cassert>
+#include <cstdint>
+#include <string_view>
+#include <optional>
+
+#include <atoms/result.hpp>
+
+#include <system/idle.hpp>
+
+#include <drivers/timer.hpp>
+#include <drivers/i2c.hpp>
+
+#include <VL53L1X_api.h>
+#include <vl53l1_platform.h>
+
+// namespace that's used to propagate needed peripherals/functions
+// to library's `vl53l1_plaftorm.cpp`
+namespace _inner {
```

Comment:
Obvyklý název je `detail`. Zároveň, všechna jména v globálním namespace, která začínají podrtžítkem jsou rezerovaná pro překladač a jejich použití je UB.